Add user documentation (#47)
nebari-dev/jupyterlab-gallery: README.md

for the server extension and a NPM package named `jupyterlab-gallery`
for the frontend extension.

When [`jupyterlab-launchpad`](https://github.com/nebari-dev/jupyterlab-launchpad) is installed, the gallery will be added as a "Gallery" section in the launcher:

![in launchpad][in-launchpad]

Otherwise it will be shown in the left sidebar.

[in-launchpad]: https://raw.githubusercontent.com/nebari-dev/jupyterlab-gallery/main/ui-tests/tests/jupyterlab_gallery.spec.ts-snapshots/in-launchpad-linux.png

## Usage

Hover over the tile with exhibit that you are interested in to reveal a "Download" button:

![hover before cloning][hover-fresh]

> Clicking this button will start the download process which you can monitor by tracking the progress bar that shows up in the tile.

Author: ```suggestion
Clicking this button will start the download process, which you can monitor by tracking the progress bar that shows up in the tile.
```

After cloning has completed, hover over the tile again to reveal "Open Folder" and "Update" buttons:

![hover after cloning][hover-cloned]

> The update buttons becomes active once new version of the cloned repository becomes available (this is once new commits are pushed to the tracked branch):

Author: ```suggestion
The update button becomes active once a new version of the cloned repository becomes available (once new commits are pushed to the tracked branch):
```
